scripts/check-package-manager-npm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script checks that the prerequisite
+# npm is installed
+
+if ! command -v npm &> /dev/null
+then
+    echo "**npm is required and not installed**"
+    echo "install Node.js LTS from:"
+    echo "https://nodejs.org/en/"
+    echo "or install and use nvm"
```

Comment:
should we also link out to [nvm](https://github.com/nvm-sh/nvm) as well?

Reply:
@AtofStryker 

`nvm` is not a prerequisite, so I don't really want to start handling that. Also there are different versions, one for Linux and a different one for Windows.

There may also be cases where an end-user wants to use the script and does **not** want to use the LTS version.

So I think it is OK the way it is.

The CI [.github/workflows/example-node-versions.yml](https://github.com/cypress-io/github-action/blob/master/.github/workflows/example-node-versions.yml) runs tests on `Node.js` `14`, `16`, `18`, `19`, so I don't think that there is any significant exposure by not checking deeper at this point.

Reply:
@MikeMcC399 I agree. I don't think it's necessary here and the jobs itself should mitigate anything if a wrong version were used. I'm not too worried about it 😄 .

Reply:
The main reason for this update was to help me from making avoidable mistakes using my own scripts! 😆